### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by AbrahamFergie**

### CODE REVIEW

**Comment by punitrathore** [comment]
I don't see the quality rubric ?

File: spec/hashTable.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+chai.use(chaiChange)
+
+describe('hashTable', () => {
+  it('exists', () => {
+    expect(HashTable).to.be.a('function')
+  })
+
+  context('put()', () => {
+    it('adds a key-value pair to the hash table.', () => {
+<<<<<<< Updated upstream
```
Comment (jrobcodes) [inline spec/hashTable.js:14]: Merge conflict artifact remaining.

File: src/hashTable.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    }
+    return sum % 31
+  }
+
+  chain(){
+    //iterate through the linkedlist until you reach a node whos next is null.
+    //set that next to the new node that is created.
+  }
+
+<<<<<<< Updated upstream
```
Comment (jrobcodes) [inline src/hashTable.js:28]: Merge conflict artifact remaining.

**Review by jrobcodes — COMMENTED** [review]
You have a few merge artifacts in your implementation file.  Otherwise looks like a good (and speedy) start.

**Pull request merged**